src/appshell/appshell.qrc

```diff
@@ -112,5 +112,6 @@
         <file>qml/Preferences/internal/MixerSection.qml</file>
         <file>qml/DevTools/Extensions/ExtensionsListView.qml</file>
         <file>qml/platform/PlatformMenuBar.qml</file>
+        <file>resources/cloud_icon.svg</file>
```

Comment:
Here is also still some redundancy that should be resolved; either this one should be used or the "installed" one